### ISSUE TITLE
Installation failing with pip on manjaro linux

### ISSUE THREAD

**Issue by sysadmin1337** [ansible/ansible-container#299]
##### ISSUE TYPE

 - Bug Report

##### OS / ENVIRONMENT

```
Manjaro Linux latest updates
Pip v9
```
##### SUMMARY

Install `sudo pip install ansible-container` output:
```
Collecting ansible-container                                                                                                                                             
  Downloading ansible-container-0.2.0.tar.gz (53kB)                                                                                                                      
    100% |████████████████████████████████| 61kB 166kB/s 
Collecting docker-compose==1.7.0 (from ansible-container)                                                                                                                
  Using cached docker-compose-1.7.0.tar.gz                                                                                                                               
Requirement already satisfied: docker-py==1.8.0 in /usr/lib/python3.5/site-packages (from ansible-container)                                                             
Requirement already satisfied: Jinja2==2.8 in /usr/lib/python3.5/site-packages (from ansible-container)                                                                  
Collecting PyYAML==3.11 (from ansible-container)                                                                                                                         
  Downloading PyYAML-3.11.zip (371kB)                                                                                                                                    
    100% |████████████████████████████████| 378kB 188kB/s 
Requirement already satisfied: requests==2.7.0 in /usr/lib/python3.5/site-packages (from ansible-container)                                                              
Requirement already satisfied: cached-property<2,>=1.2.0 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                             
Requirement already satisfied: docopt<0.7,>=0.6.1 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                                    
Requirement already satisfied: texttable<0.9,>=0.8.1 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                                 
Requirement already satisfied: websocket-client<1.0,>=0.32.0 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                         
Requirement already satisfied: dockerpty<0.5,>=0.4.1 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                                 
Requirement already satisfied: six<2,>=1.3.0 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                                         
Requirement already satisfied: jsonschema<3,>=2.5.1 in /usr/lib/python3.5/site-packages (from docker-compose==1.7.0->ansible-container)                                  
Requirement already satisfied: MarkupSafe in /usr/lib/python3.5/site-packages (from Jinja2==2.8->ansible-container)                                                      
Installing collected packages: PyYAML, docker-compose, ansible-container                                                                                                 
  Found existing installation: PyYAML 3.12                                                                                                                               
    DEPRECATION: Uninstalling a distutils installed project (PyYAML) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.                                                                                                        
    Uninstalling PyYAML-3.12:                                                                                                                                            
      Successfully uninstalled PyYAML-3.12                                                                                                                               
  Running setup.py install for PyYAML ... done                                                                                                                           
  Found existing installation: docker-compose 1.8.1                                                                                                                      
    Uninstalling docker-compose-1.8.1:                                                                                                                                   
      Successfully uninstalled docker-compose-1.8.1                                                                                                                      
  Running setup.py install for docker-compose ... done                                                                                                                   
  Running setup.py install for ansible-container ... done                                                                                                                
Successfully installed PyYAML-3.11 ansible-container-0.2.0 docker-compose-1.7.0
```

`ansible-container --help` (or any other command) result:

```
Traceback (most recent call last):
  File "/usr/bin/ansible-container", line 11, in <module>
    load_entry_point('ansible-container==0.2.0', 'console_scripts', 'ansible-container')()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 564, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2608, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2268, in load
    return self.resolve()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2274, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.5/site-packages/container/cli.py", line 213
    except exceptions.AnsibleContainerAlreadyInitializedException, e:
                                                                 ^
SyntaxError: invalid syntax
```

##### EXPECTED RESULTS
Succesfully install.

**Comment by chouseknecht** [comment]
Had not heard of manjaro linux. But yes, we are not python 3 compliant yet, so you'll need to install python 2.7. Apologies for the inconvenience.

**Comment by pilou-** [comment]
This is already fixed (4f261b97) but not released, you could try the development version:
`pip install https://github.com/ansible/ansible-container/archive/develop.zip`
however you could experience other problems related to Python 3.

**Comment by ReeSilva** [comment]
Using Arch Linux I avoid this error using `sudo pip2.7 install ansible-container`, given that as @chouseknecht said, Ansible-Container aren't ready yet for Python 3

**Comment by sysadmin1337** [comment]
@chouseknecht Is it going to be python 3 compliant eventually? @pilou- I'll try it out and report bugs back. @ReeSilva How did you install this in Arch? In what package is that command located?

**Comment by ReeSilva** [comment]
@sysadmin1337 First of all ensure that it's not installed on Python3 with `sudo pip uninstall ansible-container`.

Then, after this, install it on Python2.7 with `sudo pip2.7 install ansible-container`.

That's it.

**Comment by pilou-** [comment]
@sysadmin1337 it should be python 3 compliant now, at least test suites are executed using Python 3.5 without any error (see #300). Let us know if you encounter any problems :)

**Comment by gregdek** [comment]
Closing as fixed according to #300 -- thx @pilou- :)

**Comment by SalahAdDin** [comment]
Same problem in `python 3.6`:
```bash
ansible-container init    ✓  581  12:17:18 
Traceback (most recent call last):
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/cli.py", line 268, in __call__
    getattr(core, u'hostcmd_{}'.format(args.subcommand))(**vars(args))
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/__init__.py", line 28, in __wrapped__
    return fn(*args, **kwargs)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/core.py", line 112, in hostcmd_init
    raise AnsibleContainerAlreadyInitializedException()
container.exceptions.AnsibleContainerAlreadyInitializedException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/salahaddin/Proyectos/Works/edx-ansible/bin/ansible-container", line 11, in <module>
    sys.exit(host_commandline())
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/__init__.py", line 28, in __wrapped__
    return fn(*args, **kwargs)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/cli.py", line 270, in __call__
    logger.error('{0}'.format(e), exc_info=False)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/structlog/stdlib.py", line 81, in error
    return self._proxy_to_logger('error', event, *args, **kw)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/structlog/stdlib.py", line 119, in _proxy_to_logger
    **event_kw)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/structlog/_base.py", line 176, in _proxy_to_logger
    args, kw = self._process_event(method_name, event, event_kw)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/structlog/_base.py", line 136, in _process_event
    event_dict = proc(self._logger, method_name, event_dict)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/utils/visibility.py", line 93, in with_memoized_loggers
    return info_formatter(logger, call_name, event_dict)
  File "/home/salahaddin/Proyectos/Works/edx-ansible/lib/python3.6/site-packages/container/utils/visibility.py", line 71, in info_formatter
    sio.write(to_text(event_dict.pop('event')))
KeyError: 'event'
```
